--- origsrc/qsvn-0.8.1/src/svnqt/CMakeLists.txt	2009-01-07 05:30:07.000000000 -0600
+++ src/qsvn-0.8.1/src/svnqt/CMakeLists.txt	2009-02-18 18:30:48.699750000 -0600
@@ -167,7 +167,9 @@ ENDIF (BUILD_TESTS)
 # install rules
 # in win32 we don't install it
 IF(NOT WIN32)
-    INSTALL(TARGETS ${svnqt-name} DESTINATION ${LIB_INSTALL_DIR})
+    INSTALL(TARGETS ${svnqt-name} LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+                                  RUNTIME DESTINATION ${BIN_INSTALL_DIR}
+                                  ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
     INSTALL(FILES ${INST_HEADERS} DESTINATION include/svnqt)
     INSTALL(FILES ${CACHEINST_HEADERS} DESTINATION include/svnqt/cache)
 ENDIF(NOT WIN32)
